doc/manual/variables-and-scoping.rst

```diff
@@ -49,16 +49,20 @@ Julia uses `lexical scoping <https://en.wikipedia.org/wiki/Scope_%28computer_sci
 meaning that a function's scope does not inherit from its caller's
 scope, but from the scope in which the function was defined.
 For example, in the following code the ``x`` inside ``foo`` refers
-to the ``x`` in the global scope of its module ``Bar``::
+to the ``x`` in the global scope of its module ``Bar``:
 
-    module Bar
-    x = 1
-    foo() = x
-    end
+.. doctest::
+
+    julia> module Bar
+           x = 1
+           foo() = x
+           end;
+
+and not a ``x`` in the scope where ``foo`` is used:
 
-and not a ``x`` in the scope where ``foo`` is used::
+.. doctest::
 
-    julia> import Bar
+    julia> import Bar;
```

Comment:
Semicolon seem unnecessary: `import Bar` evaluates to `nothing`, so the same effect, no?